### DISCUSSION TITLE
Feature --silent and TPU support

### DISCUSSION THREAD

**Discussion by KyNicholas** [Purfview/whisper-standalone-win#39] [Ideas]
Currently, I am using R134.6, and I would like to inquire about any plans to add a new feature called "--silent" or something similar. Instead of displaying the entire text, it would be convenient if the program could simply show the progress as a percentage, for example, "10% completed."

Additionally, I wanted to ask the potential profitability of supporting TPU for your project, similar to "Whisper JAX." Do you think it would be beneficial to incorporate TPU support into your system?

Thank you.

**Comment by Purfview** [comment]
> it would be convenient if the program could simply show the progress as a percentage, for example, "10% completed."

It shows the progress bar already, it's in the title bar.
EDIT: or you can use `-pp` switch to show it in console.

> Additionally, I wanted to ask the potential profitability of supporting TPU for your project, similar to "Whisper JAX." Do you think it would be beneficial to incorporate TPU support into your system?

Not interested.
